optimism/src/keccak/witness.rs

```diff
@@ -179,10 +187,6 @@ impl<Fp: Field> KeccakInterpreter for KeccakEnv<Fp> {
         for (idx, value) in shifts.iter().enumerate() {
             self.write_column(KeccakColumn::SpongeShifts(idx), *value);
         }
-        let pad_blocks = pad_blocks::<Fp>(self.pad_len as usize);
```

Comment:
Could have been detected with a unit test :grimacing: :)

Reply:
Not really, as I said it makes no difference in the constraints. It's just a change for better readability and coherence.